ngirimana/murikahomeltd
# #173322618 add house listing form

> #### What does this PR do?
* add house listing form
> #### Description of Task to be completed
- add house form
- Add spinner
- fetching API for adding house
> #### How should this be manually tested?
* clone repository
* run ``` cd murika-front-end``` in terminal
* checkout  **ft-add-house-173322618**  branch by running ```git checkout ft-add-house-173322618``` in terminal
* use 'npm install' to install all necessary dependencies
* Run 'npm start' to run the server 
* And click the login link
> #### Any background context you want to provide?
* you should install react  in your PC 
> #### What are the relevant pivotal tracker stories?
* [#173322618](https://www.pivotaltracker.com/story/show/173322618)
> #### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38111965/85843397-675d3200-b7a1-11ea-9051-6eb3ec938e12.png)
